files/aws_lifecycle_hooks/attach_ebs_volume.py

```diff
@@ -59,21 +59,70 @@ def attach_volume(
             raise
 
 
+def parse_user_data():
+    # user_data = tools.get_user_data()
+    return 'a', 'b'
+
+
+def try_attach(
+        volume_id: str,
+        device_name: str,
+        instance_id: str,
+        region: str,
+        retry_limit: int,
+        retry_interval: int,
+):
```

Comment:
since you're adding type hints anyway, you can also hit the return value. In this case `-> None`